test/integration/pipeline/multisig_tx_pipeline_test.cpp

```diff
@@ -0,0 +1,172 @@
+const std::chrono::milliseconds kLoadDelay = 1000ms;
+
+class MstPipelineTest : public TxPipelineIntegrationTestFixture {
+ public:
+  void SetUp() override {
+    iroha::ametsuchi::AmetsuchiTest::SetUp();
+
+    auto genesis_tx =
+        TransactionGenerator().generateGenesisTransaction(0, {"0.0.0.0:10001"});
+    genesis_tx.quorum = 1;
+    genesis_block =
+        iroha::model::generators::BlockGenerator().generateGenesisBlock(
+            0, {genesis_tx});
+
+    manager = std::make_shared<iroha::KeysManagerImpl>("node0");
+    auto keypair = manager->loadKeys().value();
+
+    irohad = std::make_shared<TestIrohad>(block_store_path,
+                                          redishost_,
+                                          redisport_,
+                                          pgopt_,
+                                          0,
+                                          10001,
+                                          10,
+                                          kProposalDelay,
+                                          kVoteDelay,
+                                          kLoadDelay,
+                                          keypair);
+
+    ASSERT_TRUE(irohad->storage);
+
+    irohad->storage->insertBlock(genesis_block);
+    irohad->init();
+    irohad->run();
+  }
+
+  void TearDown() override {
+    iroha::ametsuchi::AmetsuchiTest::TearDown();
+    std::remove("node0.pub");
+    std::remove("node0.priv");
+    std::remove("admin@test.pub");
+    std::remove("admin@test.priv");
+    std::remove("test@test.pub");
+    std::remove("test@test.priv");
+    std::remove("multi@test.pub");
+    std::remove("multi@test.priv");
+  }
+
+  /**
+   * Creates txes for multiquorum account and send it
+   * @param account is name of new account
+   * @param signers is a list of account signers
+   * @param quorum is number of signatories required for valid tx
+   * @return true if succesful
+   */
+  bool createMultiQuorumAccount(std::string account,
+                                std::initializer_list<std::string> signers,
+                                uint32_t quorum) {
+    CommandGenerator gen;
+    auto domain = "@test";
+    auto keypair = createNewAccountKeypair(account + domain);
+    iroha::KeysManagerImpl manager("admin@test");
+    Transaction tx =
+        iroha::model::generators::TransactionGenerator().generateTransaction(
+            "admin@test",
+            1,
+            {gen.generateCreateAccount(account, "test", keypair.pubkey),
+             gen.generateSetQuorum(account, quorum)});
+    for (auto s : signers) {
+      auto key = createNewAccountKeypair(s);
+      tx.commands.push_back(gen.generateAddSignatory(s, key.pubkey));
+    }
+
+    auto admin_keypair = manager.loadKeys().value();
+    iroha::model::ModelCryptoProviderImpl(admin_keypair).sign(tx);
+    return sendForCommit(tx);
+  }
+
+  /**
+   * Perform model tx sending, and waiting for a commit
+   * @param tx is transaction for sending
```

Comment:
"transaction to be sent" sounds better. Not sure that "sending" can be used in this context.